enterprise/frontend/src/embedding-sdk/reducer.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+import { createReducer, SerializedError } from "@reduxjs/toolkit";
+import {
+  createAsyncThunk,
+  createThunkAction,
+} from "metabase/lib/redux/typed-utils";
+import type {
+  EmbeddingSessionTokenState,
+  EnterpriseState,
+  GetEnterpriseState,
+} from "embedding-sdk/types";
+
+const initialState: EmbeddingSessionTokenState = {
+  token: null,
+  loading: false,
+  error: null,
+};
+
+export const getSessionToken = (state: EnterpriseState) =>
```

Comment:
This doesn't really select the token, but rather the slice of the state about the token, this makes `const state = getSessionToken(...)` a bit confusing later on